packages/services/schema/src/api.ts

```diff
@@ -160,23 +159,25 @@ export const schemaBuilderApiRouter = t.router({
         assertAllCasesExhausted(input);
       } catch (error) {
         // Treat timeouts caused by external composition as "expected errors"
-        if (ctx.cache.isTimeoutError(error) && input.type === 'federation' && input.external) {
+        if (ctx.cache.isTimeoutError(error)) {
           return {
             errors: [
               {
-                message: error.message,
-                source: 'graphql',
```

Comment:
No idea why this was 'graphql' before. With the new guard for rejecting within `packages/services/api/src/modules/schema/providers/models/composite.ts` it makes more sense to have this be `composite`.